rubygem-deacon/rubygem-deacon.spec

```diff
@@ -0,0 +1,68 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name deacon
+
+Summary: Human readable names generator
+Name: %{?scl_prefix}rubygem-%{gem_name}
+
+Version: 1.0.0
+Release: 1%{?dist}
+Group: Development/Libraries
+License: GPLv3 and Public Domain
+URL: https://github.com/lzap/deacon
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(deacon) = %{version}
+
+%description
+Deacon ruby library provides human readable names using random (LFSR) generator
+as well as MAC-based algorithm.
+
+%package doc
+BuildArch:  noarch
+Requires:   %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+%{?scl:scl enable %{scl} "}
+gem unpack %{SOURCE0}
```

Comment:
Unpacking SOURCE0 is unnecessary if the gem_install uses %{SOURCE0} in the build section. This looks like a cross between the older method (gem_install -n %SOURCE0) and the new method (gem unpack, gem spec, gem build, gem install).

Either update the %build section to use gem build, %gem_install (not from %SOURCE0) or remove the unpacking. If you use the gem2spec template provided (scl.spec.erb) then it will use the newer method.